ui/src/pages/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Tabs } from "antd";
+import Image from "next/image";
+import { GetServerSideProps } from "next";
+
+import TabContent from "@/components/home/TabContent";
+import SearchBar from "@/components/shared/SearchBar";
+import { Journal, Country, Facets, Response } from "@/types/types";
+import { authToken, getApiUrl, getSearchUrl } from "@/utils/utils";
+
+interface HomePageProps {
+  count: number;
+  facets: Facets;
+}
+
+const HomePage: React.FC<HomePageProps> = ({ count, facets }) => {
+  const journals: Journal[] = facets?._filter_journal?.journal?.buckets;
+  const partners: Country[] = facets?._filter_country?.country?.buckets;
+
+  const tabItems = [
+    {
+      label: "Journals",
+      key: "1",
+      children: <TabContent data={journals} type="journal" />,
+    },
+    {
+      label: "SCOAP3 partners",
+      key: "2",
```

Comment:
sometimes keys are strings, sometimes int. For example in ui/src/components/shared/Header.tsx  headerItems keys are ints. Is it what we want?